examples/polkadot/analyze-jam.py

```
@@ -0,0 +1,171 @@
+    if fb < 0xf0:
+        return ((fb - 0xe0) << 24) | data[off + 1] | (data[off + 2] << 8) | (data[off + 3] << 16), 4
+    return struct.unpack_from("<Q", data, off + 1)[0], 9
+
+
+def parse_jam(path):
+    with open(path, "rb") as f:
+        data = f.read()
+
+    off = 0
+    # Metadata prefix: varint length + metadata bytes
+    meta_len, n = read_var_u32(data, off); off += n
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)